File: src/main/java/com/rs/java/game/area/banks/NardahBank.java
package com.rs.java.game.area.banks;

import com.rs.java.game.WorldTile;
import com.rs.java.game.area.Area;
import com.rs.java.game.area.Shape;
import com.rs.java.game.area.shapes.Polygon;

public final class NardahBank extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Nardah Bank";
	}

	@Override
	public Shape[] shapes() {
		return new Shape[] { new Polygon(new WorldTile[] {
				new WorldTile(3427, 2889, 0),
				new WorldTile(3427, 2894, 0),
				new WorldTile(3430, 2894, 0),
				new WorldTile(3430, 2889, 0),
				new WorldTile(3427, 2889, 0),
		}) };
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.SAFEZONE;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/banks/NardahBank.kt
package com.rs.kotlin.game.world.area.areas.banks

import com.rs.java.game.WorldTile
import com.rs.kotlin.game.world.area.Area
import com.rs.kotlin.game.world.area.shape.Polygon

class NardahBank : Area(
    Polygon(
        arrayOf(
            WorldTile(3427, 2889, 0),
            WorldTile(3427, 2894, 0),
            WorldTile(3430, 2894, 0),
            WorldTile(3430, 2889, 0),
            WorldTile(3427, 2889, 0),
        )
    )
) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Nardah Bank"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.SAFEZONE
    }
}
